site/docs/api/methods.md

```diff
@@ -496,18 +496,18 @@ The calling method syntax: `$('#table').bootstrapTable('method', parameter)`.
 
 - **Example:** [Show/Hide Loading](https://examples.bootstrap-table.com/#methods/show-hide-loading.html)
 
-## showRow
+## showRows
 
 - **Parameter:** `params`
 
 - **Detail:**
 
   Show the specified row. The params must contain at least one of the following properties:
 
-  * `index`: the row index.
-  * `uniqueId`: the value of the uniqueId for that row.
+  * `indexes`: the array of row indexes.
+  * `uniqueIds`: the array of uniqueIds for rows.
 
-- **Example:** [Show/Hide Row](https://examples.bootstrap-table.com/#methods/show-hide-row.html)
+- **Example:** [Show/Hide Rows](https://examples.bootstrap-table.com/#methods/show-hide-row.html)
```

Comment:
This would link to a example which don't show hiding multiple rows at once, could you please improve the example (https://github.com/wenzhixin/bootstrap-table-examples) ?
It would also be good if the example contains:
- show/hide/toggle one row
- show/hide/toggle multiple rows

Reply:
any news here @9r0x ?